adal/devApps/XFormsApp/SecondPage.cs

```diff
@@ -65,6 +65,10 @@ public string DrainLogs()
 
         public IPlatformParameters BrokerParameters { get; set; }
 
+        public IPlatformParameters PromptParameters { get; set; }
```

Comment:
>PromptParameters [](start = 35, length = 16)

Can't we use the normal Paramters instead? It's confusing to have all these parameters.

Reply:
yes...it's confusing, but not sure how else to get the PlatformParameter information from the specific platforms at run time. 

---
In reply to: [228564178](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1356#discussion_r228564178) [](ancestors = 228564178)

Reply:
Needs another name. Also I am not sure this should be a member of this class / i.e. it should not have a setter and the getter should just read the value from the UI. 

---
In reply to: [229936555](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1356#discussion_r229936555) [](ancestors = 229936555,229485964,228564178)